components/Images/images.css

```diff
@@ -0,0 +1,37 @@
+.img {
+  margin: var(--gap-5);
+}
+
+.img-sm {
+  width: 12rem;
+  height: 6rem;
+}
+.img-md {
+  width: 16rem;
+  height: 8rem;
+}
+
+.img-lg {
+  width: 20rem;
+  height: 10rem;
+}
+.img-fluid {
+  height: auto;
+  width: 100%;
+}
+
+.img__round--sm {
+  width: 6rem;
+  height: 6rem;
+  border-radius: 50%;
+}
+.img__round--md {
+  width: 10rem;
+  height: 10rem;
+  border-radius: 50%;
+}
+.img__round--lg {
```

Comment:
- img in shrinking in round img
use object-fit: cover; in img class ur img will not shrink

Reply:
Thanks will do the fix.